Libplanet/Store/Trie/DefaultKeyValueStore.cs

```diff
@@ -79,6 +79,15 @@ public void Dispose()
         public bool Exists(byte[] key)
             => _root.FileExists(DataPath(key));
 
+        /// <inheritdoc/>
+        public IEnumerable<byte[]> ListKeys()
+        {
+            foreach (var path in _root.EnumerateFiles(UPath.Root))
+            {
+                yield return ByteUtil.ParseHex(path.GetName());
+            }
```

Comment:
How about this?
```suggestion
        public IEnumerable<byte[]> ListKeys() =>
            _root.EnumerateFiles(UPath.Root)
                .Select(path => ByteUtil.ParseHex(path.GetName()));
```

Reply:
It looks very cool 😎  I will apply it now.